### DISCUSSION TITLE
How to install/generate link for 64bit apps on ~/scoop/shims

### DISCUSSION THREAD

**Discussion by homoioi** [ScoopInstaller/Scoop#5797] [Help and Questions]
32bit apps were installed on ~/scoop/shims

$ cd ~/scoop/shims
$ file 7z.exe 7z.shim
7z.exe: PE32 executable (console) Intel 80386, for MS Windows, 4 sections
7z.shim: ASCII text, with CRLF line terminators
$ cat 7z.shim
path = "C:\Users\zhang\scoop\apps\7zip\current\7z.exe"

$ file ../apps/7zip/current/7z.exe
../apps/7zip/current/7z.exe: PE32+ executable (console) x86-64, for MS Windows, 6 sections

So 32bit 7z under PATH of C:\Users\zhang\scoop\shims will be called. Is there any way to call the 64bit app?

Thanks,

**Comment by rashil2000** [comment]
https://github.com/ScoopInstaller/Scoop/issues/5796#issuecomment-2053929378
